tests/drivers/gpio/egpio_basic_api/testcase.yaml

```diff
@@ -0,0 +1,27 @@
+common:
+  tags:
```

Comment:
@nordic-piks, could you take a look for tags?

Reply:
Please create new tag https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/ci/tags.yaml
e.g. like here: https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/ci/tags.yaml#L1267
(name base on test path) and it to this test.
Please state dependencies for this test (at tag definition) to build this test only relevant code has changed.